cmd/parallel-manager.go

```diff
@@ -73,8 +73,8 @@ func (p *ParallelManager) addWorker() {
 	go func() {
 		for {
 			// Wait for jobs
-			fn := <-p.queueCh
-			if fn == nil {
+			fn, ok := <-p.queueCh
+			if fn == nil || !ok {
```

Comment:
why are you using `fn` before checking `ok`?

Reply:
I can use only one..